99_feature-gate.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  customNoUpgrade:
+    enabled:
+    - NodeSwap
+    - BuildCSIVolumes
```

Comment:
is the `BuildCSIVolumes` feature gate needed by `NodeSwap`?

Reply:
No, but is default enabled so I don't want to disable it.